### DISCUSSION TITLE
Use global variable

### DISCUSSION THREAD

**Discussion by IAmMrTh** [gapitio/gapit-htmlgraphics-panel#121] [Q&A] [answered]
Hey,
Is it possible to declare and use global variables in CSS?
Because  the it doesn't work with ":root".

**Comment by ZuperZee** [comment, marked as answer]
To change the _root_ of the shadow root (htmlNode) use [:host](https://developer.mozilla.org/en-US/docs/Web/CSS/:host) inside the CSS option.
![image](https://user-images.githubusercontent.com/39485579/188840916-f697f986-a9cb-4d35-b8c6-c2e909d137d8.png)

To change the root of the HTML document (entire Grafana page) use [:root](https://developer.mozilla.org/en-US/docs/Web/CSS/:root) in the Root CSS option.
![image](https://user-images.githubusercontent.com/39485579/188842046-3c543063-682d-4927-86e5-5549d8765ed1.png)

**Reply by ZuperZee** [reply]
And to use global variables in the panel use [:host](https://developer.mozilla.org/en-US/docs/Web/CSS/:root) inside the CSS option.
![image](https://user-images.githubusercontent.com/39485579/188842853-18353e64-6356-4a4b-ab20-be8919eeee31.png)

**Reply by ZuperZee** [reply]
Or use [:root](https://developer.mozilla.org/en-US/docs/Web/CSS/:root) inside the Root CSS.

![image](https://user-images.githubusercontent.com/39485579/188843222-1bf8cdb7-ca80-4add-8002-f8cfc72f9365.png)

**Reply by IAmMrTh** [reply]
Ah perfect! Thanks a lot!
